charts/timed/templates/cronjob-backend.yaml

```
@@ -0,0 +1,35 @@
+{{- range $key, $val := .Values.backend.cronjobs }}
+---
+
+apiVersion: batch/v1beta1
```

superfluous white space is superfluous
```suggestion
---
apiVersion: batch/v1beta1
```